### PR TITLE
Test account resource #16 (With latest remarks)

### PR DESCRIPTION
**Pull request by wwjfrodriguez**
Ref  [killbill-client-ruby Issue #36, PR#16](https://github.com/killbill/killbill-integration-tests/pull/16)

### CODE REVIEW

File: killbill-integration-tests/core/test_account.rb [hunk 1]
```diff
@@ -132,5 +132,54 @@ def test_account_update
...
+
+      # Verify if the returned list is empty
+      assert(blocking_states.empty?)
+
+      # Verify if response is success
+      assert(account.set_blocking_state('STATE1', 'ServiceStateService', false, false, false, nil, @user, nil, nil, @options).response .kind_of? Net::HTTPSuccess)
+
+      # Verify if the returned list has now one element
+      blocking_states = account.blocking_states('ACCOUNT', nil, 'NONE', @options)
+      assert_equal(false, blocking_states.empty?)
```
Comment (pierre) [inline killbill-integration-tests/core/test_account.rb]: You could verify the size directly:

```
assert_equal(1, blocking_states.size)
```

File: killbill-integration-tests/core/test_account.rb [hunk 2]
```diff
@@ -132,5 +132,54 @@ def test_account_update
...
+
+      # Verify blocking state fields
+      blocking_states = blocking_states.first
+      assert_equal('STATE1', blocking_states.state_name)
+      assert_equal('ServiceStateService', blocking_states.service)
+      assert_equal(false, blocking_states.block_change)
```
Comment (pierre) [inline killbill-integration-tests/core/test_account.rb]: Use `assert_false` instead (`assert_false(blocking_states.block_change)`).

**Review by pierre — APPROVED** [review]
Just a couple of small comments.

Note that you don't need to close/reopen a PR each time: if you push new commits to the same branch, the PR will automatically update.

File: killbill-integration-tests/core/test_account.rb (hunk 1 above)
Comment (wwjfrodriguez) [reply killbill-integration-tests/core/test_account.rb]: Done!

File: killbill-integration-tests/core/test_account.rb (hunk 2 above)
Comment (wwjfrodriguez) [reply killbill-integration-tests/core/test_account.rb]: Done! Thanks for the tips!

**Comment by sbrossie** [comment]
👍

**Pull request merged**